11.Font-Face/assets/css/style.css

```diff
@@ -0,0 +1,94 @@
+:root {
+  --container-width: 1200px;
+  --color-black: #000;
+  --color-grey: #dadada;
+}
+
+.konstruct-grey-bg {
+  display: inline-block;
+  background-color: var(--color-grey);
+}
+
+.konstruct-container {
+  width: 100%;
+  max-width: var(--container-width);
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.konstruct-header {
+  min-height: 80px;
+  margin-bottom: 40px;
+  display: flex;
+  align-items: center;
+}
+
+.konstruct-header .konstruct-container {
+  display: flex;
+  align-items: center;
+}
+
+.konstruct-header__language {
+  display: block;
+  padding: 2px;
+  font: 600 22px/18px Tourney, Arial, sans-serif;
```

Comment:
fill be beater to put this into separated options like font-family: 'Tourney', 'Arial'
to make it more redable and issue to debug